lib/Cake/Test/Case/Utility/HashTest.php

```diff
@@ -1361,6 +1361,38 @@ public function testSortNaturalFallbackToRegular() {
 	}
 
 /**
+ * Test sort() with locale option.
+ *
+ * @return void
+ */
+	public function testSortLocale() {
+		// get the current locale
+		$oldLocale = setlocale(LC_COLLATE, '0');
+
+		// the de_DE.utf8 locale must be installed on the system where the test is performed
+		setlocale(LC_COLLATE, 'de_DE.utf8');
```

Comment:
If the system doesn't have this locale the test will just fail, perhaps you could skip this test if `setlocale(LC_COLLATE, 'de_DE.utf8');` returns false.

From php.net/setlocale:
> Returns the new current locale, or FALSE if the locale functionality is not implemented on your platform, the specified locale does not exist or the category name is invalid.